operator/src/test/resources/application.properties

```diff
@@ -8,4 +8,6 @@ quarkus.log.level=INFO
 kc.operator.keycloak.pod-labels."test.label"=foobar
 kc.operator.keycloak.pod-labels."testLabelWithExpression"=${OPERATOR_TEST_LABEL_EXPRESSION}
 # allow the watching tests to complete more quickly
-kc.operator.keycloak.poll-interval-seconds=10
\ No newline at end of file
+kc.operator.keycloak.poll-interval-seconds=10
+# Update Pod timeout reduced to 1 min for testing
+kc.operator.keycloak.update-pod-deadline-seconds=60
```

Comment:
@shawkins @ahus1 is there a way to apply this smaller timeout when running the test suite in `remote` mode? The default "production" value is 5 min to give time to download the image, run augmentation, and run the update-compatibility command. 

Some tests use `quay.io/keycloak/non-existing-keycloak` to simulate an old image without the CLI commands and to trigger and test an image change.

Reply:
@pruivo you can set an environment variable on the operator deployment.

Reply:
It worked like a charm! Thanks @shawkins